src/selectors/balances.js

```diff
@@ -29,3 +32,28 @@ export const accountBalancesSelector = createSelector(
     return balances[activeAccountId] || {};
   },
 );
+
+export const allBalancesSelector = createSelector(
+  balancesSelector,
+  availableStakeSelector,
+  (balances, ppnBalance) => {
+    const allBalances = Object.keys(balances).reduce((memo, account) => {
+      if (balances[account] && Object.keys(balances[account]).length) {
+        const accountsBalances = Object.keys(balances[account]).map((symbol) => balances[account][symbol]);
```

Comment:
I think we could use Object.values(balances[account]) here

Reply:
P. S. if Flow complains about this, pls check our codebase for 'Object.values(' usage